packages/shared/src/libs/enums/exception-message.enum.ts

```diff
@@ -7,6 +7,7 @@ const ExceptionMessage = {
 	INVALID_TOKEN_NO_USER_ID: "Token is invalid: userId missing.",
 	NO_PERMISSION: "You do not have permission to access this resource.",
 	NO_TOKEN_PROVIDED: "No token provided.",
+	PROJECT_API_KEY_ALREADY_HAS: "Project already has its API key.",
```

Comment:
I think, we should adjust it to fit better with format 
`API key for the project not found`

Reply:
How about  `API key already exists for the project`?

Reply:
Even better!